config/jobs/kubernetes-csi/external-snapshot-metadata/external-snapshot-metadata-config.yaml

```diff
@@ -0,0 +1,433 @@
+# generated by gen-jobs.sh, do not edit manually
```

Comment:
Is this file generated by running gen-jobs.sh?  We recently updated K8s versions in gen-jobs.sh.  Please re-run it.  We should have jobs for 1.29-1.31.

Reply:
Regenerated.